doc/docs/programming_thingy.md

```diff
@@ -0,0 +1,415 @@
+   When the update is complete, a **Completed successfully** message appears.
+
+You can now disconnect the Nordic Thingy:53 from the computer and put the top cover back on.
+
+## Programming using an external debug probe
+
+You can update the Nordic Thingy application and network core firmware by using an external debug probe.
+
+The external debug probe must support Arm Cortex-M33.
+You need a 10-pin 2x5 socket-socket 1.27 mm IDC (Serial Wire Debug - SWD) JTAG cable to connect to the external debug probe.
+
+### Nordic Thingy:91 X
+
+To program Nordic Thingy:91 X using an external debug probe, complete the following steps:
+
+1. Open nRF Connect for Desktop and launch the {{app_name}}.
+1. Prepare the hardware:
+
+    a. Open the connector cover on the side of Nordic Thingy:91 X.
+
+    b. Use a JTAG cable to connect Nordic Thingy:91 X to the debug out port on a 10-pin external debug probe.
+
+    c. Power on the Nordic Thingy:91 X; move the power switch **SW1** to the **ON** position.
+
+    d. Power on the external debug probe.
+
+    e. Connect the external debug probe to the computer with a micro-USB cable.<br/>
+       In the {{app_name}}'s navigation bar, **No devices available** changes to **Select device**.
+
+     ![Programmer - Select device](./screenshots/programmer_select_device1.png "Programmer - Select device")
+
+1. Click **Select device** and select the appropriate debug probe entry from the drop-down list.<br/>
+   The icon text changes to board name and the ID of the selected device, and the **Device memory layout** section indicates that the device is connected.<br/>
+   For example, you can identify the nRF5340 DK by its PCA number PCA10095 and its ID that is printed on the label sticker on the DK.
+
+    !!! note "Note"
+        If the nRF5340 DK does not show up in the drop-down list, press Ctrl-R in Windows or command-R in macOS to restart the {{app_name}}.
+
+1. Click **Add file** in the **FILE** section, and select **Browse**.<br/>
+   A file explorer window appears.
+1. Navigate to the folder with the application firmware in the HEX format.<br/>
+   If you are programming the precompiled application firmware from the [Nordic Thingy:91 X Downloads](https://www.nordicsemi.com/Products/Development-hardware/Nordic-Thingy-91-X/Download?lang=en#infotabs), make sure to unzip the downloaded file and then open the ``img_app_bl`` folder inside.
+1. Open the folder for the application that you want to transfer to the Nordic Thingy:91 X.
+1. Select the corresponding HEX file to be used with the debug probe and click **Open**.<br/>
+   The HEX file appears in the **File memory layout** section.
+1. Click **Erase & write** in the **DEVICE** section of the side panel.
+
+The update is complete when the animation in the {{app_name}}'s **Device memory layout** section ends.
```

Comment:
Here we are missing a step there is a with to select between the 91 or 53 soc and depending on the FW we must pick the correct with position ...

Reply:
e.g. see line 305 ... we need something similar